### PR TITLE
Added separate docker image for unbundled build. Added clang-11, gcc-10 in packager image. Updated packager image to ubuntu 20.04

### PR DESCRIPTION
**Pull request by qoega**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Not for changelog

Reverted in https://github.com/ClickHouse/ClickHouse/pull/13758

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):

Added separate docker image for unbundled build. Added clang-11, gcc-10 in packager image. Updated packager image to ubuntu 20.04

Detailed description / Documentation draft:

...

By adding documentation, you'll allow users to try your new feature immediately, not when someone else will have time to document it later. Documentation is necessary for all features that affect user experience in any way. You can add brief documentation draft above, or add documentation right into your patch as Markdown files in [docs](https://github.com/ClickHouse/ClickHouse/tree/master/docs) folder.

If you are doing this for the first time, it's recommended to read the lightweight [Contributing to ClickHouse Documentation](https://github.com/ClickHouse/ClickHouse/tree/master/docs/README.md) guide first.

### CODE REVIEW

File: docker/packager/deb/Dockerfile [hunk 1]
```diff
@@ -25,68 +25,36 @@ RUN curl -O https://clickhouse-builds.s3.yandex.net/utils/1/dpkg-deb
...
-            intel-opencl-icd \
-            unixodbc-dev \
-            odbcinst \
             tzdata \
             gperf \
             alien \
-            libcapnp-dev \
             cmake \
             gdb  \
-            pigz \
```
Comment (alexey-milovidov) [inline docker/packager/deb/Dockerfile:83]: pigz is needed for packaging.

**Comment by alexey-milovidov** [comment]
Fuzzer: https://github.com/ClickHouse/ClickHouse/issues/13750

**Comment by alexey-milovidov** [comment]
Integration: #13721

**Comment by alexey-milovidov** [comment]
It does not work at all.
The reason is that "unbundled" build does not produce portable binary. And looks like the test run in different container.
No idea why the checks were green in this PR.

I will revert.
But we need to continue, we need this change.

**Comment by qoega** [comment]
>No idea why the checks were green in this PR.
I'll check what happened with docker push task between my commit b74ded3, where it was green and yours c241e42, after which stateless container failed to build. https://clickhouse-test-reports.s3.yandex.net/0/c241e42db015e990760c960a912734e529721657/push_to_dockerhub.html#fail1

**Comment by qoega** [comment]
1) The problem with stateless containers etc. is that there are only python3 versions of packages available in 20.04.
2) Actual problem with successful container build in previous check it it is not sufficient to build containers in correct order. We have to pass image tag explicitly for dependent `docker build` processes.

=> 
We need to make unbundled build without upgrade to 20.04
Transfer python2->python3 blocks ubuntu upgrade for tests containers.

**Comment by alexey-milovidov** [comment]
Ok.

**Pull request merged**